src/sql/workbench/parts/query/common/localizedConstants.ts

```diff
@@ -19,13 +19,14 @@ export const msgSaveSucceeded = localize('msgSaveSucceeded', "Successfully saved
 export const msgStatusRunQueryInProgress = localize('msgStatusRunQueryInProgress', "Executing query...");
 
 // /** Results Pane Labels */
-export const maximizeLabel = localize('maximizeLabel', "Maximize");
-export const restoreLabel = localize('resultsPane.restoreLabel', "Restore");
-export const saveCSVLabel = localize('saveCSVLabel', "Save as CSV");
-export const saveJSONLabel = localize('saveJSONLabel', "Save as JSON");
-export const saveExcelLabel = localize('saveExcelLabel', "Save as Excel");
-export const saveXMLLabel = localize('saveXMLLabel', "Save as XML");
-export const viewChartLabel = localize('viewChartLabel', "View as Chart");
+export const maximizeLabel = localize('maximizeLabel', 'Maximize');
+export const restoreLabel = localize('resultsPane.restoreLabel', 'Restore');
+export const saveCSVLabel = localize('saveCSVLabel', 'Save as CSV');
+export const saveJSONLabel = localize('saveJSONLabel', 'Save as JSON');
+export const saveExcelLabel = localize('saveExcelLabel', 'Save as Excel');
+export const saveXMLLabel = localize('saveXMLLabel', 'Save as XML');
+export const viewChartLabel = localize('viewChartLabel', 'View as Chart');
```

Comment:
>'View as Chart' [](start = 57, length = 15)

Make sure you merge in the latest changes from master - I added a hygiene check that should have caught this and prevented you from modifying these (the message text should be in double quotes)